src/elements/container/sticky-bar/sticky-bar.visual.spec.ts

```diff
@@ -80,6 +84,21 @@ describe(`sbb-sticky-bar`, () => {
       );
     });
 
+    for (const color of cases.color) {
+      it(
+        `container_color=${color}`,
+        visualDiffDefault.with(async (setup) => {
+          await setup.withFixture(
+            html` <sbb-container .color=${color}>
+              ${containerContent(color)}
+              <sbb-sticky-bar> ${actionGroup()}</sbb-sticky-bar>
```

Comment:
nitpicky, but color should also be passed to actionGroup for the sbb-block-link becoming negative